packages/x64dbgpy.vm/x64dbgpy.vm.nuspec

```diff
@@ -2,7 +2,7 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
   <metadata>
     <id>x64dbgpy.vm</id>
-    <version>1.0.56.20211021</version>
+    <version>1.0.59.20240124</version>
```

Comment:
it's not easy to find the version, since they use commit hashes for the release name, but I trust this is accurate

Reply:
I think it is, I found this from the ci pipeline